beheer.md

```diff
@@ -23,3 +24,65 @@ Bij de overdracht van een standaard van ontwikkeling naar beheer moeten ook de n
 Aangezien bij overdracht van ontwikkeling naar beheer de GitHub repository as-is wordt overgenomen kan de backlog zelf blijven waar deze is.
 
 Overdracht van de backlog en GitHub repository is een officieel moment en moet plaatsvinden voordat er sprake kan zijn van in beheername van een Open API
+
+## Werken met Docker
+
+Zie https://docs.docker.com/get-started/ en specifiek https://docs.docker.com/get-started/part2/ en https://docs.docker.com/get-started/part3/ voor een diepere uitleg over Docker.
+
+Voor het vervaardigen van Docker images heb je het volgende nodig:
+
+* een goed werkende Docker installatie;
+* een folder met daarin:
+  - een Dockerfile (met de naam 'Dockerfile');
+  - een requirements file (met de naam requirements.txt);
+  - een python applicatie;
+  - evt. een 'docker-compose.yml' file.
+* een account op 'hub.docker.com'
+
+>_Hebben we ook authorisatie nodig op hub.docker.com om de docker files in de juiste repository weg te kunnen schrijven?_
```

Comment:
Ter info: De repository van een component bevat alle code van de applicatie + een Dockerfile.

Een `built` van deze Docker pushen we naar Dockerhub.

Voor zowel de repo als Dockerhub heb je rechten nodig.